Makefile

```diff
@@ -34,16 +34,20 @@ docs: ## generate Sphinx HTML documentation, including API docs
 	tox -e docs
 	$(BROWSER) docs/_build/html/index.html
 
+# Define PIP_COMPILE_OPTS=-v to get more information during make upgrade.
+PIP_COMPILE = pip-compile --rebuild --upgrade $(PIP_COMPILE_OPTS)
```

Comment:
I think we only want to have `--rebuild` in the first pip-compile call, not in all of them.

Reply:
This is copied from the cookiecutter: https://github.com/edx/edx-cookiecutters/blob/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/Makefile#L32-L46